### ISSUE TITLE
[BUG] Directory specified in PAPERLESS_TRASH_DIR doesn't get created on Container start

### ISSUE THREAD

**Issue by WERTBON1789** [paperless-ngx/paperless-ngx#4605]
### Description

Currently the trash directory specified in PAPERLESS_TRASH_DIR doesn't seem to get created automatically.

![image](https://github.com/paperless-ngx/paperless-ngx/assets/50809022/2fd22afa-91b0-4c76-9bcf-bcf525325601)

The documentation states that it should be put in a permanent volume, media and data are the same volume in my case so I would think that even if I used "../media/trash" this issue would still occur.

### Steps to reproduce

Try to put the trash dir in a custom location which isn't a volume itself

### Webserver logs

```bash
Running Django checks
SystemCheckError: System check identified some issues:

ERRORS:
?: PAPERLESS_TRASH_DIR is set but doesn't exist.
	HINT: Create a directory at ../data/trash/

System check identified 1 issue (0 silenced).
```

### Paperless-ngx version

1.17

### Host OS

Flatcar Container Linux

### Installation method

Docker - official image

### Configuration changes

PAPERLESS_TRASH_DIR=../data/trash/

**Comment by stumpylog** [comment]
I'm not sure why we'd create a directory that is under control of the user?

**Comment by stumpylog** [comment]
Yeah, the more I look at this, if you've set PAPERLESS_TRASH_DIR, you've defined it to something and should create that something.  If you'd like to use the suggestion, create that folder.  If you'd like another, create that one.  In either case, it feels up to the user to ensure the folder exists, is writeable, and is somewhere which will be saved for a Docker installation.

**Comment by WERTBON1789** [comment]
Well, if I'm following your argument, why are the PAPERLESS_DATA_DIR and PAPERLESS_MEDIA_ROOT directorys created?
isn't it the user's task to ensure the directory exists, is writeable, and is somewhere which will be saved for a Docker installation?
Also, the documentation not only suggests that I can put the media directory to the same path as the data directory, it also suggests that I can put the trash directory in the media directory, so it would only make sense if it is created just like the other ones

**Comment by stumpylog** [comment]
The data directory and media directory must exist to function.  The trash directory is an optional directory

**Comment by WERTBON1789** [comment]
Fair enough, but from a usability standpoint, why wouldn't you just check if the trash directory exists and create it if it doesn't?
Like I said, I just pulled the settings I use from the documentation and they don't really work with Docker.
Also I'm using a normal named volume, not a bind mount to a directory and I want to install the whole thing as unattended as possible, so what I did now is to overwrite docker-prepare.sh to include a check if the directory exists and if it doesn't not, it gets created, but I would really prefer a built-in solution for that and most stuff actually behaves like that (e.g. Docker itself) so I just expected it to do that
